rm.go

```diff
@@ -144,9 +141,18 @@ func removeContainer(ctx context.Context, client *containerd.Client, id string,
 		}
 	default:
 		if !force {
-			return errors.Errorf("cannot remove a %v container %v", status.Status, id)
+			var msg string
+			if status.Status == containerd.Paused || status.Status == containerd.Running {
```

Comment:
Can we move these comparison to `switch`?

Reply:
Fixed.